keyserver/api/api.go

```diff
@@ -53,9 +53,17 @@ func (k *KeyError) Error() string {
 	return k.Err
 }
 
+type DeviceMessageType int
+
+const (
+	TypeDeviceKeyUpdate DeviceMessageType = iota
+	TypeCrossSigningUpdate
+)
+
 // DeviceMessage represents the message produced into Kafka by the key server.
 type DeviceMessage struct {
-	DeviceKeys
+	Type        DeviceMessageType `json:"Type,omitempty"`
```

Comment:
Capital T?

Reply:
Pretty much because none of the fields had JSON key name annotations before, so they were capitalised as the field names were written. This is just in-keeping with that.